packages/plugin-persistent-cache/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@vitejs/plugin-persistent-cache",
+  "version": "4.1.1",
+  "license": "MIT",
+  "author": "Guillaume Chau",
+  "files": [
+    "dist"
+  ],
+  "keywords": [
+    "frontend",
+    "vite",
+    "vite-plugin",
+    "cache"
+  ],
+  "main": "./dist/index.cjs",
```

Comment:
I would suggest adding `"type": "module"` and only distributing an ESM version since this is a new plugin. Vite is working on dropping the CJS versions of its builds - they're deprecated in v5 and will be removed in v6